src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -79,18 +79,18 @@ export const InitializeClient: FC<OwnProps> = ({
 
   useEffect(() => {
     dispatch(getBuckets())
-  }, []) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [dispatch])
 
   useEffect(() => {
     onSelectBucket(bucket.name)
   }, [bucket, onSelectBucket])
 
   useEffect(() => {
-    const fetchResources = async () => {
-      await dispatch(getAllResources())
```

Comment:
Same here - safe to remove `await` because it isn't being used to hold up any other code.

Reply:
follow up - if the `async/await` is removed, then there's no point to call `dispatch(getAllResrouces())` inside the `fetchAllResources` function. Remove `fetchAllResources` and just call `dispatch(getAllResrouces())` in the body of the hook's anonymous function

Reply:
@alexjw3 In this PR, could you also make the same cleanup to `Tokens.tsx`, which has equivalent code (no need for a helper function for `dispatch`, since we're not `await`ing it anymore? Thanks!

Reply:
Makes sense.